### ISSUE TITLE
Feature Request: Show nicks/usernames list on Left Hand Side

### ISSUE THREAD

**Issue by Aligorith** [prawnsalad/KiwiIRC#927]
It would be nice to have a toggle (or even better, a persistent preference) to show the users list on the Left Hand Side of the screen, instead of having it always show on the right hand side. After years of using Chatzilla, I always find it weird seeing this list on the right, so it'd be nice to have an option to configure this.

**Comment by Arkadietz** [comment]
Yes, i am agreed with that one. Or as i said in my previous statement will be good if you can choose using variable in the config.
